web/themes/custom/cloudy/cloudy.theme

```diff
@@ -114,13 +114,17 @@ function cloudy_form_search_api_page_block_form_search_portland_gov_alter(&$form
   /* merge returned search class */
   $form['#attributes']['class'] = array_merge($form['#attributes']['class'], $additional_classes);
 
-  $text = t("Search Portland.gov");
+  if ($is_front_page) {
+    $text = t("Search Portland.gov");
+  } else {
+    $text = t("Search");
```

Comment:
I think we might want to  keep this as "Search portland.gov" for the rest of the site as well. Bureaus think of themselves as islands and the sitewide search pages like News and Events have a second "search" form. Making it clear that search is all of Portland.gov makes more sense in the navigation context. 🤔